### ISSUE TITLE
Error building example project

### ISSUE THREAD

**Issue by kashira-m** [bevyengine/bevy#4215]
## Bevy version 
0.6.0

## Operating system & version

macOS Big Sur 11.6,4

## What you did

https://bevyengine.org/learn/book/getting-started/
I tried to run example following the instruction.

## What you expected to happen

Example runs successfully

## What actually happened

Build error while trying to build breakout(example).

## Additional information

log is here

cargo run --example breakout
   Compiling winit v0.26.1
   Compiling gilrs-core v0.3.2
   Compiling gltf-json v0.16.0
   Compiling hexasphere v6.1.0
   Compiling coreaudio-sys v0.2.9
error: failed to run custom build command for `coreaudio-sys v0.2.9`

Caused by:
  process didn't exit successfully: `/Users/yu/projects/bevy/target/debug/build/coreaudio-sys-73a049c00a254662/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=COREAUDIO_SDK_PATH
  cargo:rustc-link-lib=framework=AudioUnit
  cargo:rustc-link-lib=framework=CoreAudio
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS

  --- stderr
  coreaudio.h:1:10: fatal error: 'AudioUnit/AudioUnit.h' file not found
  coreaudio.h:1:10: fatal error: 'AudioUnit/AudioUnit.h' file not found, err: true
  thread 'main' panicked at 'unable to generate bindings: ()', /Users/yu/.cargo/registry/src/github.com-1ecc6299db9ec823/coreaudio-sys-0.2.9/build.rs:145:39
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed

**Comment by DJMcNab** [comment]
Based on the error message, it looks like your issue is related the audio support. It suggests you need to install `CoreAudio`?

According to https://developer.apple.com/documentation/coreaudio, `CoreAudio` should be available for 'macOS 10.0+', although there doesn't seem to be a download link on that page.

As a workaround, you should be able to just disable the audio feature. Unfortunately, since cargo doesn't have a way to subtract features afaik, I think you need to use a command which lists out all the features individually:

```shell
$ cargo run --example breakout --no-default-features --features render,bevy_winit
```

**Comment by x-52** [comment]
The `CoreAudio` framework comes with macOS (it's a system API). However, the header files needed to link against it aren't installed by default. To get them, you need the Xcode Command Line Tools, which you can get by running
```sh
$ xcode-select --install
```
You don't need Xcode to install these tools, by the way. It also contains a C compiler and linker, both of which are needed to build the `coreaudio-sys` crate.
